examples/hif2a/fit_to_multiple_rbfes.py

```diff
@@ -447,33 +410,31 @@ def equilibrate_edges(datasets: List[Dataset], systems: List[Dict[str, Any]], nu
                     blown_up = True
                     print(f"step {step} blew up in {stage} stage")
 
-            # note: unflatten_grad and unflatten_theta have identical definitions for now
-            flat_loss_grad, unflatten_grad = flatten(loss_grads, ordered_handles)
-            flat_theta, unflatten_theta = flatten(ordered_params, ordered_handles)
+            flat_loss_grad = flatten(loss_grads)
+            flat_theta = flatten(ordered_params)
 
             # based on current estimate of (loss, grad, and simulation stability), return a conservative step to take in parameter space
             theta_increment = truncated_step(flat_theta, loss, flat_loss_grad,
                                              step_lower_bound=_compute_step_lower_bound(loss, blown_up))
-            param_increments = unflatten_theta(theta_increment)
+            param_increments = unflatten(theta_increment)
 
             # for any parameter handler types being updated, update in place
-            for handle in ordered_handles:
+            for (handle, increment) in zip(ordered_handles, param_increments):
```

Comment:
nit: the `()` around handle and increment are unncessary

Reply:
Addressed in https://github.com/proteneer/timemachine/pull/474/commits/98a5653f4a6584744dd136abd196e4c32b5ed386